src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/IWcfService.cs

```diff
@@ -65,5 +66,14 @@ internal interface IWcfService
 
         [OperationContract(Action = "http://tempuri.org/IWcfService/GetDataUsingXmlSerializer"), XmlSerializerFormat]
         XmlCompositeType GetDataUsingXmlSerializer(XmlCompositeType composite);
+
+        [OperationContract]
+        Stream GetStream(string data);
```

Comment:
NIT: Consider naming this `GetStreamFromString`, as the meaning of this endpoint will eventually become lost as we add more tests in. Same with the `UploadStream` - consider naming something like `GetStringFromStream`